.github/scripts/branch_pr_issue_closer.py

```diff
@@ -135,176 +98,162 @@ def _main_prog():
 
-        #Split text into individual words:
-        post_msg_word_list = post_msg_text.split()
+    for pr_num in pr_nums:
 
-        #Extract first word:
-        first_word = post_msg_word_list[0]
+        #+++++++++++++++++++++++++++++++++++++
+        #Check that PR has in fact been merged
+        #+++++++++++++++++++++++++++++++++++++
 
-        #Print merged pr number to screen:
-        print(f"Merged PR: {first_word}")
+        #Extract pull request info:
+        merged_pull = cam_repo.get_pull(pr_num)
 
-        try:
-            #Try assuming the word is just a number:
-            pr_num = int(first_word[1:]) #ignore "#" symbol
-        except ValueError:
-            #If the conversion fails, then this is likely not a real PR merge, so end the script:
-            endmsg = "No Pull Request number was found in the commit message, so there is nothing for the script to do."
+        #If pull request has not been merged, then exit script:
+        if not merged_pull.merged:
+            endmsg = f"Pull request associated with commit:\n{trigger_sha}\n"
+            endmsg += "was not actually merged, so the script will not close anything."
             end_script(endmsg)
 
-    else:
-        endmsg = "This push commit does not appear to be a merged pull request, so the script will do nothing."
-        end_script(endmsg)
-
-    #+++++++++++++++++++++++++++++++++++++
-    #Check that PR has in fact been merged
-    #+++++++++++++++++++++++++++++++++++++
+        #++++++++++++++++++++++++++++++++++++++++
+        #Check that PR was not for default branch
+        #++++++++++++++++++++++++++++++++++++++++
 
-    #Extract pull request info:
-    merged_pull = cam_repo.get_pull(pr_num)
+        #Determine default branch on repo:
+        default_branch = cam_repo.default_branch
 
-    #If pull request has not been merged, then exit script:
-    if not merged_pull.merged:
-        endmsg = "Pull request in commit message was not actually merged, so the script will not close anything."
-        end_script(endmsg)
-
-    #++++++++++++++++++++++++++++++++++++++++
-    #Check that PR was not for default branch
-    #++++++++++++++++++++++++++++++++++++++++
-
-    #Determine default branch on repo:
-    default_branch = cam_repo.default_branch
-
-    #Extract merged branch from latest Pull request:
-    merged_branch = merged_pull.base.ref
+        #Extract merged branch from latest Pull request:
+        merged_branch = merged_pull.base.ref
 
-    #If PR was to default branch, then exit script (as github will handle it automatically):
-    if merged_branch == default_branch:
-        endmsg = "Pull request ws merged into default repo branch. Thus issue is closed automatically"
-        end_script(endmsg)
+        #If PR was to default branch, then exit script (as github will handle it automatically):
+        if merged_branch == default_branch:
+            endmsg = "Pull request was merged into default repo branch. "
```

Comment:
Might be nice to include the PR number
```suggestion
            endmsg = f"Pull request #{pr_num} was merged into default repo branch. "
```

Reply:
Yes!  PR number added.